src/Servers/IIS/AspNetCoreModuleV2/CommonLib/HostFxr.h

```diff
@@ -26,9 +26,13 @@ class HostFxrErrorRedirector: NonCopyable
     static inline thread_local RedirectionOutput* m_writeFunction;
 };
 
-class HostFxr
+class HostFxr: NonCopyable
 {
 public:
+    HostFxr() : HostFxr(nullptr, nullptr, nullptr)
+    {
```

Comment:
Why do we need this?

Reply:
To set default value of HostFxr inside CLR thread context.